MiniProject1-master/Holmes_Mini_Project_1.py

```diff
@@ -0,0 +1,89 @@
+from pattern.en import sentiment
+from pattern.en import modality
```

Comment:
You can import multiple things simultaneously, e.g.
`from pattern.en import sentiment, modality`